packages/editor-ui/src/views/NodeView.vue

```diff
@@ -2231,7 +2235,15 @@ export default mixins(
 				}
 
 				if(node.type === STICKY_NODE_TYPE) {
-					this.$telemetry.track('User deleted workflow note', { workflow_id: this.$store.getters.workflowId });
+					this.$telemetry.track(
+						'User deleted workflow note',
+						{
+							workflow_id: this.$store.getters.workflowId,
+							is_welcome_note: node.name === QUICKSTART_NOTE_NAME,
+							instance_id: this.$store.getters.instanceId,
+							user_id: this.$store.getters.currentUserId,
```

Comment:
don't need to add `instance_id` and `user_id` it's included by default in telemetry

Reply:
Didn't know that. Will remove.